docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: "3"
```

Comment:
can you make a directory for this file and other files used by docker compose, e.g. under `deploy/docker-compose`?

Reply:
Ok will do. I will also update the read me.

Reply:
cool, looking forward to the change